telethon/client/users.py

```diff
@@ -26,10 +26,12 @@ def _fmt_flood(delay, request, *, early=False, td=datetime.timedelta):
 
 
 class UserMethods:
-    async def __call__(self: 'TelegramClient', request, ordered=False):
+    async def __call__(self: 'TelegramClient', request, ordered=False, flood_sleep_threshold=None):
         return await self._call(self._sender, request, ordered=ordered)
```

Comment:
```suggestion
        return await self._call(self._sender, request, ordered=ordered, flood_sleep_threshold=flood_sleep_threshold)
```

`flood_sleep_threshold` not passed to `self._call`, why?